### PR TITLE
Fix in tabbed with alpha patch

### PR DESCRIPTION
**Pull request by XPhyro**
This patch fixes sxiv crashing when attempting to embed in windows with transparency.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -24,7 +24,7 @@ lib_exif_0 =
 lib_exif_1 = -lexif
 lib_gif_0 =
 lib_gif_1 = -lgif
-ldlibs = $(LDLIBS) -lImlib2 -lX11 -lXft -lfontconfig \
+ldlibs = $(LDLIBS) -lImlib2 -lX11 -lXft -lfontconfig -lXrender\
```
Comment (TAAPArthur) [inline Makefile:27]: Is this needed? I don't see any new includes or use of Xrender in general.
And if it is needed, going to object to the new mandatory dependency.

Comment (XPhyro) [reply Makefile:27]: It seems to compile and work fine without, so I do not know why the author included it. Maybe somebody else has some more information?

Comment (TAAPArthur) [reply Makefile:27]: If this is the entire patch and there are no missing files/commits, then the above diff isn't needed. It simply isn't used. The only thing it does as is to make the build failed in Xrender isn't installed. Other than this line, the change looks fine to me.

Comment (XPhyro) [reply Makefile:27]: This is the whole patch.

To keep the author's patch as-is, I'll add a commit on top of this to remove the dependency.

**Review by TAAPArthur — APPROVED** [review]

**Review by eylles — APPROVED** [review]

**Comment by eylles** [comment]
for this one do you think it is better to squash into one commit or leave it as 2 commits?

**Comment by N-R-K** [comment]
Squash.

**Comment by N-R-K** [comment]
Where exactly is this PR from? I was assuming it was an open PR under sxiv repo, but I can't quite find anything there. @XPhyro 

This patch introduces the [following](https://github.com/nsxiv/nsxiv/pull/112#issuecomment-936798925) regression

> It wouldn't even start when the server didn't support TrueColor. The original sxiv did work in that case, but drew a black box around transparent images instead of the background color.

**Comment by XPhyro** [comment]
@N-R-K https://github.com/muennich/sxiv/issues/342#issuecomment-759555974

**Pull request merged**